### PR TITLE
(on getgov-za) Ticket #1990: Update text styling on sticky bar

### LINKED ISSUE #1990: Update text styling on sticky bar
### Issue description

During demos we had this feedback: "On the domain request sticky bar make "Requested domain: " smaller and not bolded. Then make the domain name bolded."

### Acceptance criteria

- [ ] domain name is bolded
- [ ] Requested domain: is smaller and not bold

### Additional context

_No response_

### Links to other issues

_No response_

### PR DESCRIPTION
**Pull request by zandercymatics**
## Ticket

Resolves #1990

## Changes

- Modifies the styling on _admin.scss
- Modifies the html on domain_request_change_form.html
- Modifies existing unit test

## Context for reviewers
This PR makes the sticky bar on Domain Admin > Domain Request a bit more pleasing to look at

## Setup
Repeat these steps as a superuser and an analyst:
1. Navigate to /admin
2. Navigate to Domain Request
3. Click a record
4. Note that the "Requested domain" portion of the sticky bar is now:
- Not bolded for requested domain, and bolded for the domain name
- A bit smaller

#### Satisfied acceptance criteria and met development standards

- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the associated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the associated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [x] Checked that the design translated visually
- [x] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [x] Checked for landmarks, page heading structure, and links
- [x] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [x] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [x] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [x] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

File: src/registrar/assets/sass/_theme/_admin.scss [hunk 1]
```diff
@@ -495,6 +495,9 @@ address.dja-address-contact-list {
         overflow: hidden;
         white-space: nowrap;
         text-overflow: ellipsis;
+        // This has to be set here due to style overrides
```
Comment (rachidatecs) [inline src/registrar/assets/sass/_theme/_admin.scss:498]: Nitpick - no need for the comment

**Review by rachidatecs — APPROVED** [review]

**Review by therealslimhsiehdy — APPROVED** [review]
**Previously:**
Screenshot 2024-04-16 at 10 53 28 AM

**After:**
Screenshot 2024-04-16 at 10 54 26 AM

I think per the AC, this matches great and LGTM! 

I don't know if this is a big concern, but thought it was worth a possible mention. For shorter urls this loads and displays great! However, if the url is a bit longer, it seems to cut out `.gov` or ends of the URL. 
Screenshot 2024-04-16 at 10 53 50 AM
Screenshot 2024-04-16 at 10 54 15 AM

**Comment by gabydisarli** [comment]
Hey @zandercymatics can you ping me when this is back on your sandbox? I will be doing the design review.

**Review by gabydisarli — APPROVED** [review]
looks great!

**Pull request merged**